Improve the Learn Page Descriptions (#2242)
ballerina-platform/ballerina-dev-website: learn.md

<h2>Tooling Guide</h2>

> <p>Learn about using the VSCode plugin and other CLI tools.</p>

Reviewer: ```suggestion
<p>Learn to use the VSCode plugin and other CLI tools.</p>
```

Author: Fixed.
